### ISSUE TITLE
Fiber web server rebooting twice per file change

### ISSUE THREAD

**Issue by danawoodman** [makiuchi-d/arelo#3]
With a basic Fiber web server, I'm seeing the web server get rebooted twice on change of `*.go` files with this command:

```shell
arelo -p '**/*.go' -i '**/.*' -i '**/*_test.go' -- go run .
```

![image](https://github.com/makiuchi-d/arelo/assets/157695/c1772fe4-18bb-4293-b050-12bc372fcc12)

**Comment by danawoodman** [comment]
My guess is the edit from VSCode is triggering a `CHMOD` event from what I'm seeing when watching fsnotify locally

**Comment by makiuchi-d** [comment]
It seems like two events (the one might be a `CHMOD`) are occurring with an interval of more than 1 second.
Please try increasing the --delay (-d) option.

```
arelo -p '**/*.go' -i '**/.*' -i '**/*_test.go' -d 3s -- go run .
```

**Comment by danawoodman** [comment]
Ok, but then that slows down the restart loop. Maybe an `-f CHMOD` flag of some kind like npm's `onchange` to omit that event from being listened to?

**Comment by makiuchi-d** [comment]
@danawoodman 
I have released v1.13.0, which supports the `-f` option.
Please try it!
